minikube/bootstrap.sh

```diff
@@ -116,147 +116,106 @@ waitDashboardIsDeployed() {
   set -e
 }
 
-minikubeVer(){
-LATEST_VER=`curl -s https://github.com/kubernetes/minikube/releases/latest| sed -e 's/.*v\(.*\)".*/\1/'`
-
-LATEST_VER_SHA=`curl -s -L https://github.com/kubernetes/minikube/releases/download/v$VER/minikube-linux-$ARCH.sha256 -o \
-    /tmp/minikube.sha && cat /tmp/minikube.sha`
-
-CURRENT_VER=`sha256sum /usr/local/bin/minikube | awk '{print $1}'`
-
-if ! [[ "$LATEST_VER_SHA" == "CURRENT_VER" ]]; then
-echo "curl -s -LO https://github.com/kubernetes/minikube/releases/download/v$VER/minikube-linux-$ARCH"
-fi
+#helmLinuxUpgrade check new version and upgrade helm using version number
+helmLinuxUpgrade() {
+  unset LATEST_VER CURRENT_VER SHA CHECK_VER
+  LATEST_VER=`curl -s https://github.com/kubernetes/helm/releases/latest | sed -e 's/.*v\(.*\)".*/\1/'`
```

Comment:
Use `https://api.github.com/repos/kubernetes/helm/releases/latest` to deal with JSON structured data, not HTML.

See [example](https://github.com/instrumentisto/gitlab-builder-docker-image/blob/980045487eea0aadb930ecdfea0666b8fe6b9aa7/Dockerfile#L46-L50).